plugins/modules/notification/slack.py

```diff
@@ -214,6 +230,8 @@
 OLD_SLACK_INCOMING_WEBHOOK = 'https://%s/services/hooks/incoming-webhook?token=%s'
 SLACK_INCOMING_WEBHOOK = 'https://hooks.slack.com/services/%s'
 SLACK_POSTMESSAGE_WEBAPI = 'https://slack.com/api/chat.postMessage'
+SLACK_UPDATEMESSAGE_WEBAPI = 'https://slack.com/api/chat.update'
+SLACK_CONVERSATION_HISTORY_WEBAPI = 'https://slack.com/api/chat.update'
```

Comment:
I see SLACK_CONVERSATION_HISTORY_WEBAPI is the same value as SLACK_UPDATEMESSAGE_WEBAPI. I'm not very familiar with the Slack API at this point, but I see that there is an actual endpoint called `conversation.history` https://api.slack.com/methods/conversations.history so I wonder if the intention was to use that endpoint at first, but for some reason the `chat.update` made more sense?

Reply:
no, your assumption is correct, I just forgot to update this, probably while figuring out what to name the variable.